app/validators/dependent_field_inclusion_validator.rb

```diff
@@ -2,11 +2,14 @@ class DependentFieldInclusionValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     parent_field_name = options[:parent]
     parent_field_value = record.attributes[parent_field_name]
-    parent_field_value_lookup = parent_field_value.to_s.downcase
-    mapping = options[:in]
 
-    valid_values = mapping.dig(parent_field_name, parent_field_value_lookup).map(&:downcase)
-    return if value.downcase.in?(valid_values)
+    unless parent_field_value.nil?
```

Comment:
I think Rubocop normally complains about `unless` in a multiline construct as a large chunk of code can be predicated on a double negative.

```suggestion
    if parent_field_value
```